tests/clar.c

```diff
@@ -440,21 +446,26 @@ void
 clar_test_shutdown(void)
 {
 	clar_print_shutdown(
+		_clar.evaluation_duration,
 		_clar.tests_ran,
-		(int)_clar_suite_count,
-		_clar.total_errors
+		_clar.suites_ran,
+		_clar.total_errors,
+		_clar.total_skipped
 	);
 
 	clar_unsandbox();
 }
 
 int
-clar_test(int argc, char **argv)
+clar_test(int argc, char *argv[])
 {
+	(int)argc;
+	(char**)argv;
 	int errors;
+	double start = git__timer();
```

Comment:
I still think this should actually be part of clar_test_run, as otherwise the timer will only ever work when clar_test is being called.